src/main/java/com/brandon3055/draconicevolution/common/utills/InventoryUtils.java

```diff
@@ -7,11 +7,20 @@
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemStack;
 
+import baubles.common.lib.PlayerHandler;
+
 public final class InventoryUtils {
 
     private InventoryUtils() {}
 
     public static Optional<ItemStack> getItemInPlayerInventory(EntityPlayer player, Class<? extends Item> item) {
+
+        Optional<ItemStack> baublesSlots = Arrays.stream(PlayerHandler.getPlayerBaubles(player).stackList)
+                .filter(itemStack -> itemStack != null && item.isInstance(itemStack.getItem())).findAny();
+        if (baublesSlots.isPresent()) {
+            return baublesSlots;
+        }
```

Comment:
Move Baubles-specific logic into separate method and call it only when Baubles mod is loaded.